crates/bevy_pbr/src/render/mesh_view_bindings.rs

```diff
@@ -201,7 +203,11 @@ fn layout_entries(
     layout_key: MeshPipelineViewLayoutKey,
     render_device: &RenderDevice,
     render_adapter: &RenderAdapter,
-) -> Vec<BindGroupLayoutEntry> {
+) -> [Vec<BindGroupLayoutEntry>; 2] {
+    // EnvironmentMapLight
```

Comment:
Is this comment here for Ctrl+F users or

Reply:
It just moved when i moved the block, will delete.